### DISCUSSION TITLE
API PUT for heating setPoint

### DISCUSSION THREAD

**Discussion by sjpbailey** [tagyoureit/nodejs-poolController#908] [Nixie] [answered]
Hello,
 I have started with the Nixie controller Sequent Mega-BAS and eight relay hats. 
 I am coding in Python a new Plugin formally a Node server for Universal Devices, plan on all Bodies for Plugins and AqualinkD. 
 I have my circuits functional and overridable with PUT however heating setPoint is giving me NULL when i do a PUT to it. 
version @ 8.0.1

`headers = {
    'accept': 'application/json',
    # Already added when you pass json=
    'Content-Type': 'application/json',
}

json_data = {
    "id": 1,
    "name": "Pool",
    "setPoint": '75',
}

response = requests.put(
    'http://192.168.1.53:4200/state/body/setPoint', headers=headers, json=json_data)`

**Comment by tagyoureit** [comment, marked as answer]
Best thing to do is load up dashPanel and make the same call and view how it is structured in the browser developer console.

**Reply by sjpbailey** [reply]
Thank You for YOUR time!
DOH, lol "now i see", said the blind man "all the put commands from the browser".
